adafruit_display_text/label.py

```
@@ -248,13 +256,7 @@ def _update_text(
             i = 0
         tilegrid_count = i
 
-        try:
```

I think we may need to bring back this try / except. 

Attempting to use `terminalio.FONT` (simpletest script) I am getting this error with this version from this PR:
```
Traceback (most recent call last):
  File "code.py", line 7, in <module>
  File "/lib/adafruit_display_text/label.py", line 135, in __init__
  File "/lib/adafruit_display_text/label.py", line 259, in _update_text
  File "/lib/adafruit_display_text/label.py", line 192, in _get_ascent
  File "/lib/adafruit_display_text/label.py", line 181, in _get_ascent_descent
AttributeError: 'BuiltinFont' object has no attribute 'load_glyphs'
```

---

I had the right idea in needing to catch an attribute error, but wrong place.  Up inside of `_get_ascent_descent` was the correct place to catch and ignore this error since the built-in font doesn't use that.